lib/app/modules/us_core_r4/pediatric_bmi_for_age_sequence.rb

```diff
@@ -217,9 +217,58 @@ def validate_resource_item(resource, property, value)
         validate_history_reply(@observation, versioned_resource_class('Observation'))
       end
 
-      test 'Observation resources associated with Patient conform to Argonaut profiles' do
+      test 'Demonstrates that the server can supply must supported elements' do
         metadata do
           id '10'
+          link 'https://build.fhir.org/ig/HL7/US-Core-R4/general-guidance.html/#must-support'
+          desc %(
+          )
+          versions :r4
+        end
+
+        must_support_elements = [
+          'Observation.status',
+          'Observation.category',
```

Comment:
does it matter that `Observation.category` is listed twice?

Reply:
Ah, probably has to do with the `category` slice.  Did you explicitly take into account slices or is that a potential gap?

Reply:
It's a gap. I didn't know how to handle slices so I just left them there for now.